code/modules/mob/living/carbon/carbon.dm

```diff
@@ -492,4 +492,7 @@
 
 /mob/living/carbon/flash_eyes(intensity = FLASH_PROTECTION_MODERATE, override_blindness_check = FALSE, affect_silicon = FALSE, visual = FALSE, type = /obj/screen/fullscreen/flash)
 	if(eyecheck() < intensity || override_blindness_check)
-		return ..()
\ No newline at end of file
+		return ..()
+
+/mob/living/carbon/get_contained_external_atoms()
+	. = contents - internal_organs
```

Comment:
Does Aurora not have `organs`?

Reply:
Ah nvm this is carbon.

Reply:
yeah i dunno, it's weird. we gotta get rid of carbon one day